status_api/README.md

```diff
@@ -0,0 +1,48 @@
+# NVSS API Status App (API and UI Build)
+
+## Dependencies
+ - .NET Core 6.100
+ - Docker with MSSQL Server
+
+## Quick Start
+
+1. The status_api uses models, migrations, and database from `messaging`. Follow its [README](../README.md) to get started
+and run a few API calls to populate its database.
```

Comment:
It might be helpful to point the user to the top level testing_tools directory and its README.md for populating the database for testing the status ui.